internal/operators/cnv/manifest.go

```diff
@@ -57,6 +63,16 @@ func Manifests(config Config) (map[string][]byte, []byte, error) {
 
 	openshiftManifests := make(map[string][]byte)
 
+	if shouldInstallHPP(config, cluster) {
+		cnvHpp, err := hpp(config.SNOPoolSizeRequestHPPGib)
```

Comment:
can/should users override that? is that what is going to be used or is it going to take full disk capacity?

Reply:
Users can override this:
https://github.com/openshift/assisted-service/blob/68d297e4e44585c71d71a28990fab3a0a7d96aae/internal/operators/cnv/config.go#L16-L17
But they should not. If they want persistent VMs (wide use case) on SNO, they likely want a solution like HPP to allow them.
We are requesting `SNOPoolSizeRequestHPPGib` but get attached to the full LSO disk PV which can be bigger.

Reply:
If thats only the minimum then you  probably should add a comment explaining that

Reply:
This is not specific to this situation; it has to do with how k8s will match a PVC with a PV
`(Kubernetes will match PVs with the PVC created from this template if the PV capacity >= request size)`